appeals/web/src/server/lib/mappers/appeal/has.js

```diff
@@ -22,7 +22,6 @@ import { mapCostsLpaCorrespondence } from './submappers/costs-lpa-correspondence
 import { mapCostsLpaWithdrawal } from './submappers/costs-lpa-withdrawal.mapper.js';
 import { mapCrossTeamCorrespondence } from './submappers/cross-team-correspondence.mapper.js';
 import { mapDecision } from './submappers/decision.mapper.js';
-import { mapFinalCommentReviewDueDate } from './submappers/final-comment-review-due-date.mapper.js';
```

Comment:
Are the mappers you removed from here redundant now?

Reply:
I've realigned the timetable so we have these deadlines:

-       lpaQuestionnaireDueDate?: string | null;
- 	ipCommentsDueDate?: string | null;
- 	appellantStatementDueDate?: string | null;
- 	lpaStatementDueDate?: string | null;
- 	appellantFinalCommentsDueDate?: string | null;
- 	lpaFinalCommentsDueDate?: string | null;
- 	s106ObligationDueDate?: string | null;
- 	issueDeterminationDate?: string | null;

and realigned the mappers accordingly... I noticed that we only showing the lpaStatementDueDate, not the appellantStatement one...